src/components/typewriter/Typewriter.tsx

```diff
@@ -7,6 +7,24 @@ import { GameStatus } from '@/pages';
 
 const TIMEOUT_DURATION = 300;
 
+const E1 = Math.floor(Math.random() * 5 + 1);
+const E2 = Math.floor(Math.random() * 9 + 1);
+const E3 = Math.floor(Math.random() * 8 + 1);
+const E4 = Math.random()
+    .toString(36)
+    .replace(/[^a-z]+/g, ``)
+    .substr(0, 1);
+const E5 = Math.floor(Math.random() * 6 + 1);
+const E6 = Math.random()
+    .toString(36)
+    .replace(/[^a-z]+/g, ``)
+    .substr(0, 1);
+const E7 = Math.random()
+    .toString(36)
+    .replace(/[^a-z]+/g, ``)
+    .substr(0, 1);
+const E8 = Math.floor(Math.random() * 6 + 1);
+
```

Comment:
DRY.
Const name does not indicate its purpose.
All those randomized script can be written within 2 lines. 
`substr()` is Deprecated
If we will have a random game state it will look smth like this: